### DISCUSSION TITLE
Searx is Rubbish!

### DISCUSSION THREAD

**Discussion by SunnyTime4Me** [searx/searx#3471] [Public Searx instances: report a bug]
Ԝhу ⅾоո't уοս сսոtѕ ⅿаkе уοսr ѕhіt ѕеаrсh еոցіոе ԝοrk рrοреrⅼу?
![Searx on Searx](https://user-images.githubusercontent.com/125903289/220153940-cfdbd24a-6c00-4214-962d-e32cf8afa65b.JPG)

![Searx on Google](https://user-images.githubusercontent.com/125903289/220154222-3d0e36b5-2cba-4db8-a26e-a3072de86da6.JPG)

**Comment by unixfox** [comment]
You should try to use another instance that work directly: https://searx.space/

**Comment by glanham-jr** [comment]
I have a local instance running and did not run into your issue. Luckily, your screenshot provided what public instance you were using.  I highly suspect your issue is occurring because the instance you were using is on quite an old version, 0.15, and the up to date version is 1.1.0. 

I would suggest finding a public instance with a later version before calling the maintainers names for no good reason.
